lightning/src/routing/router.rs

```diff
@@ -1101,7 +1101,9 @@ where L::Target: Logger {
 		} }
 	}
 
-	let empty_node_features = NodeFeatures::empty();
+	let mut onion_format_optional_features = NodeFeatures::empty();
+	onion_format_optional_features.set_variable_length_onion_optional();
```

Comment:
I'd say use a method for this so it can be reused in the tests and name the variable `default_node_features`. No need to have the name talk about the feature itself. FWIW, I think we lose some meaning by dropping the "variable length" part of it, but we can avoid that by just using a generic name.